build.gradle

```diff
@@ -141,6 +141,8 @@ dependencies {
 		exclude module: 'guava'
 		exclude module: 'gson'
 	}
+
+	compile group: 'com.carrotsearch', name: 'hppc', version: '0.7.1'

```

Comment:
100% my fault, read a series of performance graphs backwards. Will remove HPPC later today.